site/_scss/blocks/_toc.scss

```diff
@@ -30,6 +30,35 @@
   transform: rotate(270deg);
 }
 
+toc-active {
+  display: block;
+  max-height: calc(100vh - #{get-size(200)} * 2);
+  overflow-x: hidden;
+  overflow-y: auto;
+  position: sticky;
+  top: get-size(200);
+
+  &::-webkit-scrollbar {
+    height: 8px;
+    width: 8px;
+  }
+  &::scrollbar {
+    height: 8px;
+    width: 8px;
+  }
```

Comment:
Kind of wondering if we should reuse the scrollbar mixin and style from web.dev

https://github.com/GoogleChrome/web.dev/blob/main/src/styles/utilities/_scrollbars.scss
https://github.com/GoogleChrome/web.dev/blob/main/src/styles/tools/_mixins.scss#L65

Reply:
I've mirrored them for now.